daemon/src/device.rs

```diff
@@ -2790,6 +2931,15 @@ impl<'a> Device<'a> {
                     .await;
                 self.settings.save().await;
             }
+            GoXLRCommand::SetTapTempoWindow(duration_ms) => {
+                // Clamp to a sensible range (1000ms..=2000ms)
+                let clamped: u16 = duration_ms.clamp(1000, 2000);
```

Comment:
Assuming this is kept, I'm not a fan of clamping invalid values, if the value is wrong, it should be rejected and not transparently changed under the hood. It's simply not clear to users when they change something and it's not set to the value they sent.